src/block/posts/design.js

```diff
@@ -38,5 +38,37 @@ const list = ( output, style, attributes, comps ) => {
 	)
 }
 
+// Vertical 2
+const invertedVertical = ( output, style, attributes, comps ) => {
+	if ( style !== 'inverted-vertical' ) {
+		return output
+	}
+
+	const {
+		contentOrder = [],
+	} = attributes
+
+	// If style is vertical, put the image wrapper outside of the article tag.
+	const contents = contentOrder.map( key => {
+		if ( key === 'featured-image' ) {
+			return null
+		}
+
+		const comp = comps[ key ]
+		return comp
+	} )
+
+	return (
+		<>
+			<article className="stk-container-padding">
+				{ compact( contents ).map( content => content ) }
+			</article>
+			{ comps[ 'featured-image' ] }
+		</>
+	)
+}
+
```

Comment:
What does this do? It looks like without this, the inverted vertical works as intended